modules/gltf/gltf_document.cpp

```diff
@@ -477,6 +478,40 @@ String GLTFDocument::_gen_unique_name(Ref<GLTFState> state, const String &p_name
 	return name;
 }
 
+String GLTFDocument::_sanitize_animation_name(const String &name) {
+	// Animations disallow the normal node invalid characters as well as  "," and "["
+	// (See animation/animation_player.cpp::add_animation)
+
+	// TODO: Consider adding invalid_characters or a _validate_animation_name to animation_player to mirror Node.
+	String p_name = _sanitize_scene_name(name);
+	RegEx regex("([\\[,]+)");
```

Comment:
I'd suggest doing a plain `.replace()` for the invalid characters instead of using `RegEx`. The regex module can be disabled, and here this would trigger compilation errors if you don't guard it with `#ifdef MODULE_REGEX_ENABLED` (that's a pre-existing issue in this class though).

Reply:
Done, thanks!
I left the regex handling in `_sanitize_bone_name` but can do a followup if we decide to relax sanitization there as well.